ppr-ui/src/utils/mhr-api-helper.ts

```diff
@@ -168,3 +168,23 @@ export async function searchMhrPDF (searchId: string): Promise<any> {
       }
     })
 }
+
+export async function submitMhrRegistration (data) {
+  const url = sessionStorage.getItem('MHR_API_URL')
+  const config = { baseURL: url, headers: { Accept: 'application/json' } }
+
+  try {
+    const result = await axios.post('registrations', data, config)
+    if (!result?.data) {
+      throw new Error('Invalid API response')
+    }
+    return result.data
+  } catch (error) {
```

Comment:
Throwing `Invalid API response` error above would be caught here. Intentional?

Reply:
that's what we did in `searchMhrPDF`, I wanted to throw an actual `Error` instead of just returning a object with a property called error.